### PR TITLE
UI Translated to Traditional Chinese

### PR DESCRIPTION
**Pull request by 3anology**
繁體中文(台灣)
Translated added message to Traditional Chinese

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [x] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by xiuqi08** [comment]
> 繁體中文(台灣)
> Translated added message to Traditional Chinese
> 
> * [x]  Bugfix (non-breaking change which fixes an issue)
> * [ ]  New feature (non-breaking change which adds functionality)
> 
> ## Proposed changes
> ## Checklist
> * [x]  I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
> * [x]  For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
> * [x]  I have run `grunt` to verify the unit tests pass
> * [x]  I have added suitable unit tests to cover the new/changed functionality

I confirmed the Traditional Chinese part and only found one place need to be modified.
It's a typo on row 789
Before：您可以將流程的json檔拖入編輯方塊或 {{core:show-import-dialog}} 來導入流程。
Modified：您可以將流程的json文檔拖入編輯方塊或 {{core:show-import-dialog}} 來導入流程。

**Comment by knolleary** [comment]
@3anology thank you for this and thanks to @xiuqi08 for reviewing.

@3anology are you able to make the change suggested above?

**Comment by 3anology** [comment]
Thank you,
Fixed line 789 and uploaded a new node translation file

**Comment by 3anology** [comment]
@knolleary 
suggestions done, ready to merge, please help merge

**Comment by liuderchi** [comment]
@knolleary Thanks for your time reviewing this 🙇‍♂️
Please help merge it for better UX for Chinese users, thank you.

File: packages/node_modules/@node-red/editor-client/locales/zh-TW/infotips.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "tip3": "您可以在 {{core:manage-palette}} 中管理節點的控制台。",
+        "tip4": "側邊欄中會列出流程中所有的配置節點。您可以通過功能表或者 {{core:show-config-tab}} 來訪問這些節點。",
+        "tip5": "您可以在設定中選擇顯示或隱藏這些提示。",
+        "tip6": "您可以用[left] [up] [down] [right]鍵來移動被選中的節點。按住[shift]可以更快地移動節點。",
+        "tip7": "把節點拖到連接上可以向連接中插入節點。",
+        "tip8": "您可以用 {{core:show-export-dialog}} 來匯出被選中的節點或標籤頁中的流程。",
+        "tip9": "您可以將流程的json文字檔拖入編輯方塊或 {{core:show-import-dialog}} 來導入流程。",
+        "tip10": "按住[shift]後按一下並拖動節點可以將該節點的多個連接一併移動到其他節點的埠。",
+        "tip11": "{{core:show-info-tab}} 可以顯示「資訊」標籤頁。 {{core:show-debug-tab}} 可以顯示「調試」標籤頁。",
+        "tip12": "按住[ctrl]的同時點擊工作介面可以在節點的對話欄中快速添加節點。",
```
Comment (JiyeYu) [inline packages/node_modules/@node-red/editor-client/locales/zh-TW/infotips.json]: Hi, I find a type here.
工作介面 should be 工作界面

**Comment by kazuhitoyokoi** [comment]
@knolleary @dceejay As I said today in person, after merging this pull request, my colleague @JiyeYu can add translations to complete it and @xiuqi08 can able to review them as double-checking.

**Comment by knolleary** [comment]
Thank you @kazuhitoyokoi (and everyone else!)

**Pull request merged**